modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_undo_manager.scss

```diff
@@ -1,7 +1,3 @@
-.undomanager .lfr-undo-queue {
-	margin: 10px auto;
-}
-
```

Comment:
Seems like the removed classes are still in use in https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/undo_manager.js

Reply:
They might be replaceable with utilities

Reply:
yeah!